src/lifx/packets/stateDeviceChain.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+const Packet = {
+  size:
+    1 +
+    16 * (2 + 2 + 2 + 2 + 4 + 4 + 1 + 1 + 1 + 4 + 4 + 4 + 8 + 8 + 2 + 2 + 4) +
+    1,
```

Comment:
The `size` is quite confusing like this. Just use the end result value like on the other packets.

Reply:
I think I get what you're saying @ristomatti. Since those are the UDP groupings from LIFX's docs, usually I put those segments in a comment. Something like:

```js
size: X // https://docs.lifx.com/some-udp-article
```

Or even:

```js
size: X // This is a grouping of 2, 2, 2, 2, 4, 4, 1, ... etc groupings from LIFX's UDP protocol.
```

Reply:
I link to the relevant section of the LAN protocol specification would be sufficient. We don't want to duplicate the documentation here as it occasionally also changes.

Some time ago LIFX has published a machine readable spec document that should make it possible to generate parts of the code. That might be another angle to look into if we'd like to get away with these relatively cryptic packet files. The naming follows the names used in the spec document though so it's quite easy to figure out each part.

Reply:
I simply changed it to the value.
To put such things in the comments is not helpful.
So we lose the source of this arbitrated value, i can live with this.